.github/workflows/controllo_link.yml

```diff
@@ -0,0 +1,63 @@
+name: Broken Links Checker
+on:
+  push:
+    branches:
+      - master
+  schedule:
+     - cron: '0 0 1 * *'
+env:
+  ISSUE_TEMPLATE: ".github/workflows/ISSUE_TEMPLATE.md"
+ 
+  contents: read
+  issues: write 
+  
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Get links from README.md
+      id: get-links
+      run: |
+       LINKS=$(grep -oP '\[.*?\]\((http[s]?://[^)]*)\)' README.md | sed -E 's/\[.*\]\(([^)]+)\)/\1/' | paste -sd "," -)
+       echo "WEBSITE_URL=$LINKS" >> $GITHUB_ENV
+      
+    - name: Run Broken Links Checker
+      run: |
+        # Dividi la lista di link in un array separato da virgola
+        IFS=',' read -ra LINKS <<< "$WEBSITE_URL"
+        BROKEN_LINKS=""
+
+        # Scansiona i link e verifica se sono raggiungibili
+        for link in "${LINKS[@]}"; do
+          if [[ $link =~ ^(http|https):// ]]; then
+            # Se il link inizia con "http://" o "https://", verifica se è raggiungibile
+            if ! curl -IsSk "$link" > /dev/null; then
```

Comment:
Do we want `-k` here?

Also we should probably keep the output, so we can check what's the specific problem in the action logs, if we want more details.